packages/cli/src/lib/config/config.ts

```diff
@@ -22,16 +24,10 @@ export interface Configuration {
   anonymous?: boolean | undefined;
 }
 
-export const DefaultConfig: Configuration = {
-  environment: DEFAULT_ENVIRONMENT,
-  region: DEFAULT_REGION,
-  organization: '',
-  analyticsEnabled: undefined,
-  accessToken: undefined,
-  anonymous: undefined,
-};
+class IncompatibleConfiguration extends Error {}
```

Comment:
Maybe put this custom error in a different file?

Reply:
Because at first it was just a instanceof checker I was not convinced, but because I'll add a message, yeah, could make sense